Added course booking with or without patient user

skueppa opened the pull request.

siebertm commented on index.php:
```diff
@@ -142,10 +206,14 @@
-            Termin-Info: <?= $result->data->name ?> <?= $result->data->starts_at ?><br/>
-            <?php } ?>
+            <?php } else { 
+						if (!array_key_exists ( 'success' , $result )) {?>
```
thanks sören, 

could you change your editor settings to use a 2-spaces indentation instead of tabs, then re-save all the files in the 2nd commit and push again ;)

skueppa commented:
updated commit "added possibility to book courses without patient user (without OAuth)":
1. no tabs occure
2. corrected a line where php-tag wasn't closed correctly

skueppa commented:
- added courses booking with patient user
- no tabs anymore?

rousisk commented:
Yes, just waiting for a commit msg change to reflect the new work on OAuth
